src/audio/rtnr/rtnr_stub.c

```diff
@@ -36,11 +37,16 @@ void RTKMA_API_Prepare(void *Context)
 
 void *RTKMA_API_Context_Create(int sample_rate)
 {
-	return NULL;
+	/* Allocate something, to avoid return NULL and cause error
+	 * in check of success of this.
+	 */
+	return rzalloc(SOF_MEM_ZONE_RUNTIME, 0, SOF_MEM_CAPS_RAM, 42);
```

Comment:
is 42 == sizeof(audio_stream_rtnr)? if so, better use sizeof, or if number are random, you can use 4, 42 is not easy to read.

Reply:
I can make it a macro. We don't know the real size used by the closed library, and the run fails if this is NULL. Since I had to choose something, number 42 a reference to plot  of https://en.wikipedia.org/wiki/The_Hitchhiker%27s_Guide_to_the_Galaxy . :)

Reply:
you may need move 42 definition from:
https://github.com/thesofproject/sof/pull/8301/files#diff-f09311b76861c69cd17f987de389037705c0964aaf78d0d72e0e6da39bcdb2ffR12

to a more common place and reuse it between rtnr and igo_nr module.

Reply:
this can be done later, now, I am ok go for merge.